resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml

```diff
@@ -182,7 +163,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: gcr.io/knative-releases/github.com/knative/eventing/cmd/controller:v0.4.1
+        image: gcr.io/knative-releases/github.com/knative/eventing/cmd/controller:{{ .Values.version }}
```

Comment:
Do you think it is worth to parametrise version? New version means new YAMLs so you'll be overwriting this file anyway probably. If you think it is necessary add note to README